ReactSkia/sdk/OnScreenKeyBoard.cpp

```diff
@@ -19,13 +19,22 @@ namespace sdk {
 
 std::mutex oskLaunchExitCtrlMutex;//For synchronized OSK Launch & Exit
 
+OnScreenKeyboard::OnScreenKeyboard() {
+#if ENABLE(FEATURE_KEY_THROTTLING)
+    /*create Queue for Repeat key Processing */
+    sem_init(&sigKeyConsumed_, 0, 0);
+    repeatKeyQueue_ =  std::make_unique<ThreadSafeQueue<rnsKey>>();
+    repeatKeyHandler_ = std::thread(&OnScreenKeyboard::repeatKeyProcessingThread, this);
```

Comment:
repeatKeyProcessingThread will be alive even when OSK is not in use .
Should we create it when launched and kill when OSK exits.

Reply:
I'm using ThreadSafeCache, so it will be waiting on pop queue, when OSK exit happens or no key available to process. Also that is the reason, couldn't close the thread, as it won't finish